tests/Command/Issue/IssueShowCommandTest.php

```diff
@@ -17,55 +17,86 @@
     {
         $tester = $this->getCommandTester(new IssueShowCommand());
         $tester->execute(['issue' => 60]);
 
         $this->assertCommandOutputMatches(
             [
-                'Issue #60 (open): by weaverryan [cordoval]',
-                'Milestone: v1.0',
-                'Labels: actionable, easy pick',
-                'Title: Write a behat test to launch strategy',
+                'Issue #60 - Write a behat test to launch strategy by weaverryan [open]',
+                'Org/Repo: gushphp / gush',
                 'Link: https://github.com/gushphp/gush/issues/60',
+                'Labels: actionable, easy pick',
+                'Milestone: v1.0',
+                'Assignee: cordoval',
                 'Help me conquer the world. Teach them to use Gush.',
             ],
             $tester->getDisplay()
         );
     }
 
-    public function testShowsIssueWithNumberFromBranchName()
+    /**
+     * @test
+     */
+    public function it_shows_issue_with_number_from_branch()
     {
         $command = new IssueShowCommand();
         $tester = $this->getCommandTester(
             $command,
             null,
             null,
             function (HelperSet $helperSet) {
-                $helperSet->set($this->getLocalGitHelper('test_branch')->reveal());
+                $helperSet->set($this->getLocalGitHelper('60-something-something')->reveal());
             }
         );
 
         $tester->execute();
 
         $this->assertCommandOutputMatches(
             [
-                'Issue #60 (open): by weaverryan [cordoval]',
-                'Milestone: v1.0',
-                'Labels: actionable, easy pick',
-                'Title: Write a behat test to launch strategy',
+                'Issue #60 - Write a behat test to launch strategy by weaverryan [open]',
+                'Org/Repo: gushphp / gush',
                 'Link: https://github.com/gushphp/gush/issues/60',
+                'Labels: actionable, easy pick',
+                'Milestone: v1.0',
+                'Assignee: cordoval',
                 'Help me conquer the world. Teach them to use Gush.',
             ],
             $tester->getDisplay()
         );
     }
 
-    private function getLocalGitHelper()
+    /**
+     * @test
+     */
+    public function it_errors_when_the_number_cannot_be_auto_determined()
+    {
+        $command = new IssueShowCommand();
+        $tester = $this->getCommandTester(
+            $command,
+            null,
+            null,
+            function (HelperSet $helperSet) {
+                $helperSet->set($this->getLocalGitHelper('something')->reveal());
+            }
+        );
+
+        $this->setExpectedException(
+            'Gush\Exception\UserException',
+            'Unable to extract issue-number from the current branch.'
+        );
+
+        $tester->execute();
+    }
+
+    private function getLocalGitHelper($branch = 'test_branch')
     {
-        $helper = $this->getGitHelper(true);
```

Comment:
this one is going to fail with the revisions i made, not sure yet how to tweak it